Python/ceval.c

```diff
@@ -1753,6 +1752,9 @@ _PyEval_EvalFrameDefault(PyThreadState *tstate, _PyInterpreterFrame *frame, int
     _Py_CODEUNIT *first_instr;
     _Py_CODEUNIT *next_instr;
     PyObject **stack_pointer;
+#ifdef LLTRACE
+    int lltrace;
```

Comment:
This should be `int lltrace = 0;` and the declaration should go above `if (throwflag)` above.

The `generator.throw()` case is what is what's causing the excessive output in tests, I believe.

Reply:
Done.